### PR TITLE
8322321: Add man page doc for -XX:+VerifySharedSpaces

### PR DESCRIPTION
**Pull request by jmtd**
This is an unclean backport of a manpage fix, for Oracle parity. The conflict was trivial to resolve manually and was due to the troff source being updated post-17u for 8297164, which is unlikely to be backported itself.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8322321](https://bugs.openjdk.org/browse/JDK-8322321) needs maintainer approval

### Issue
 * [JDK-8322321](https://bugs.openjdk.org/browse/JDK-8322321): Add man page doc for -XX:+VerifySharedSpaces (**Enhancement** - P4 - Approved)

### Reviewers
 * [Aleksey Shipilev](https://openjdk.org/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk17u-dev.git pull/2098/head:pull/2098` \
`$ git checkout pull/2098`

Update a local copy of the PR: \
`$ git checkout pull/2098` \
`$ git pull https://git.openjdk.org/jdk17u-dev.git pull/2098/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2098`

View PR using the GUI difftool: \
`$ git pr show -t 2098`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk17u-dev/pull/2098.diff">https://git.openjdk.org/jdk17u-dev/pull/2098.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk17u-dev/pull/2098#issuecomment-1876835794)

### CODE REVIEW

**Review by shipilev — APPROVED** [review]
Looks okay.

**Comment by shipilev** [comment]
Nope, you now need maintainer approval. My review only covers the "unclean backport" part.

**Comment by jmtd** [comment]
> Nope, you now need maintainer approval. My review only covers the "unclean backport" part.

Gotcha, thanks. I'm somewhat rusty. This is useful [grist for the mill](https://fosdem.org/2024/schedule/event/fosdem-2024-2460-a-beginner-s-guide-to-backports/).

**Comment by GoeLin** [comment]
Hi @jmtd 
somewhat off-topic: you might want to have a look at "dependent pull requests", i.e. how to stack several PRs.
There is  a current example: PRs 105, 129, 130.  Have a look at the branch they compare to (openjdkd:pr/129 etc).

**Comment by GoeLin** [comment]
For this change, you should have a look at the broken GHA tests and tell in the PR what is going on.
From the content of the change it's pretty likely that that is unrelated stuff, but these should be checked by the backporter. I will only approve changes with green GHA tests, or, if red, if the backporter states the that the issues are unrelated to his change.

**Comment by GoeLin** [comment]
I'll remove the label for now.

**Comment by GoeLin** [comment]
Hi @jmtd, have you given this up?

**Comment by jmtd** [comment]
Hi @GoeLin no, I just haven't had a chance to pivot back to backports this week. Sorry! I'll look at it Monday.

**Comment by jmtd** [comment]
THe GHA test failure macos-x64 (hs/tier1 gc) is unrelated (gc/logging/TestMetaSpaceLog.java                                    Error.
Agent error: java.lang.Exception: Agent 7 timed out with a timeout of 480
seconds; check console log for any additional details).

**Comment by jmtd** [comment]
/approval request Oracle parity, docs improvement. GHA test failure (macos-x64 hs/tier1 gc) unrelated. TIA!

**Comment by GoeLin** [comment]
Thanks!

**Pull request closed without merging**
